CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 ## 6.45.0 (Unreleased)
 
+NOTES:
+* iam: Update service account creation to prevent failures due to eventual consistency (reverted) ([#23583](https://github.com/hashicorp/terraform-provider-google/pull/23583))
+DEPRECATIONS:
+* gemini: deprecated the `disable_web_grounding` field in the `google_gemini_gemini_gcp_enablement_setting` resource ([#23581](https://github.com/hashicorp/terraform-provider-google/pull/23581))
+
+FEATURES:
+* **New Resource:** `google_bigtable_schema_bundle` ([#23585](https://github.com/hashicorp/terraform-provider-google/pull/23585))
+* **New Resource:** `google_compute_preview_feature` ([#23631](https://github.com/hashicorp/terraform-provider-google/pull/23631))
+* **New Resource:** `google_dialogflow_cx_generator` ([#23605](https://github.com/hashicorp/terraform-provider-google/pull/23605))
+* **New Resource:** `google_model_armor_floorsetting` ([#23621](https://github.com/hashicorp/terraform-provider-google/pull/23621))
+* **New Resource:** `google_vertex_ai_endpoint_with_model_garden_deployment` ([#23632](https://github.com/hashicorp/terraform-provider-google/pull/23632))
+
+IMPROVEMENTS:
+* accesscontextmanager: added `name` to `google_access_context_manager_gcp_user_access_binding` resource ([#23638](https://github.com/hashicorp/terraform-provider-google/pull/23638))
+* apigee: marked the field `access_logging_config` immutable in `google_apigee_instance` resource ([#23571](https://github.com/hashicorp/terraform-provider-google/pull/23571))
+* bigquery: added `ignore_auto_generated_schema` virtual field to `google_bigquery_table` resource to ignore server-added columns in the `schema` field ([#23633](https://github.com/hashicorp/terraform-provider-google/pull/23633))
+* cloudrunv2: promoted field `node_selector` in `google_cloud_run_v2_job` resource to GA ([#23586](https://github.com/hashicorp/terraform-provider-google/pull/23586))
+* compute: added `params.resourceManagerTags` field to the `google_compute_subnetwork` ([#23618](https://github.com/hashicorp/terraform-provider-google/pull/23618))
+* compute: added `rule.match.src_secure_tags`, `rule.target_secure_tags`, `predefined_rules.match.src_secure_tags` and `predefined_rules.target_secure_tags` fields to `google_compute_firewall_policy_with_rules` resource ([#23635](https://github.com/hashicorp/terraform-provider-google/pull/23635))
+* dataproc: added `cluster_config.security_config.identity_config` field to `google_dataproc_cluster` resource ([#23613](https://github.com/hashicorp/terraform-provider-google/pull/23613))
+* dataproc: updated `cluster_config.gce_cluster_config.metadata` field to be computed in `google_dataproc_cluster` resource ([#23613](https://github.com/hashicorp/terraform-provider-google/pull/23613))
```

Comment:
This looks like update the field from optional to optional + compute. This change is probably to fix a permadiff. Looks like a bug fix to me